### DISCUSSION TITLE
anyone know how to request something to be removed

### DISCUSSION THREAD

**Discussion by Shluffy** [Scratch-Bookmarklets/Scratch-Bookmarklets.github.io#80] [Q&A] [answered]
for all-contributors

**Comment by Shluffy** [comment]
also are there any possible things to be added that i missed (like at first I missed content)
@FunctionalMetatable

**Comment by FunctionalMetatable** [comment, marked as answer]
[heres all the possible contributions](https://allcontributors.org/docs/en/emoji-key). you cannot request something to be removed, but you can manually remove it

**Comment by Shluffy** [comment]
@all-contributors please add @FunctionalMetatable for Answering Questions

**Reply by Shluffy** [reply]
also test to see if this works
